src/containers/util.rs

```diff
@@ -49,20 +48,12 @@ pub struct TerminalDisclose {
 )]
 pub struct Terminal {
     pub seals: SmallOrdSet<XChain<TerminalSeal>>,
-    pub witness_tx: Option<XPubWitness>,
 }
 
 impl Terminal {
     pub fn new(seal: XChain<TerminalSeal>) -> Self {
         Terminal {
             seals: small_bset![seal],
-            witness_tx: None,
```

Comment:
Why do we remove the witness_tx here?

Reply:
Since now it is present directly in the `SealWitness` structure for all bundles, not just terminal ones.